airflow/auth/managers/models/resource_details.py

```diff
@@ -46,7 +46,7 @@ class DagDetails:
 class AssetDetails:
     """Represents the details of an asset."""
 
-    uri: str | None = None
+    id: str | None = None
```

Comment:
I think this will break AF2 model, but this will be deleted soon. 

@vincbeck just need to validate the auth manager changes.

Reply:
should we keep both uri and id for now?

Reply:
I'm not sure actually, I think Vincent will know what's the best way to move forward.